pkgs/top-level/aliases.nix

```diff
@@ -321,6 +321,8 @@ mapAliases {
 
   cq-editor = throw "cq-editor has been removed, as it use a dependency that was disabled since python 3.8 and was last updated in 2021"; # Added 2024-05-13
 
+  calligra = kdePackages.calligra; # Added 2024-09-27
```

Comment:
These are roughly alphabetical. Roughly. But still.

Reply:
*roughly* is putting in a lot of work there, haha! I added it above `callP...`.